kratos/geometries/geometry_data.h

```diff
@@ -769,46 +776,28 @@ class GeometryData
     ///@name Member Variables
     ///@{
 
-    SizeType mDimension;
+    GeometryDimension const* mpGeometryDimension;
```

Comment:
why do you need a pointer here?

Reply:
At some geometries I will store the static member of the Geometry Dimension, to not copy it everytime. It is the same concept as GeometryData is stored within Geometry and the derived objects.

Reply:
I think @RiccardoRossi is referring to changing the const pointer with a reference which has the same behavior.

Reply:
I cannot make it a reference beacuse I cannot initialize it with default values in the constructor. So I keep it as pointer if that's ok?

Reply:
ok